jena-text/src/main/java/org/apache/jena/query/text/TextIndexLucene.java

```diff
@@ -704,7 +704,7 @@ private String composeQField(String qs, String textField, String lang, boolean u
             if (this.isMultilingual && StringUtils.isNotEmpty(lang) && !lang.equals("none")) {
                 textField += "_" + lang;
             }
-            textClause = textField + ":" + qs + " ";
+            textClause = textField + ": (" + qs + ") ";
```

Comment:
Does the `usingSearchFor` branch of the `if` also need this fix?

Reply:
Yes, I minimized the config, so it no longer has multilingual search active, which was the original use-case. Will add.